food-truck-api/src/main/java/food/truck/api/recommendation/PassiveGuestRecommendationStrategy.java

```diff
@@ -0,0 +1,62 @@
+package food.truck.api.recommendation;
+
+import food.truck.api.Position;
+import food.truck.api.truck.Truck;
+import food.truck.api.truck.TruckService;
+import food.truck.api.user.UserPreferences;
+import org.springframework.data.util.Pair;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class PassiveGuestRecommendationStrategy implements TruckRecommendationStrategy {
```

Comment:
I was pretty sure Kevin had basically already implemented a passive guest strategy in the original `GuestRecommendationStrategy` (since that one didn't require the user to fill out a form to get recommendations, it was based solely off of truck ratings, etc). Was there an issue with what he'd implemented?